pyscraper/sp/resolvenames.py

```diff
@@ -136,25 +138,24 @@ def match_string_somehow(self,s,date,party,just_name):
 
         if not just_name:
 
-            office_matches = self.officeslowered.get(s.lower())
+            office_name = s.replace('The ', '')
+            office_matches = self.offices.get(office_name)
             if office_matches:
                 for o in office_matches:
-                    if date and date < o['start_date'] or date > o['end_date']:
+                    if date and date < o['start_date'] or 'end_date' not in o.keys() or date >= o['end_date']:
```

Comment:
I think this needs some brackets; in python `if A and B or C` returns C if A is false, rather than False.